crates/holochain-services/Cargo.toml

```diff
@@ -0,0 +1,19 @@
+[package]
+name = "holochain-services"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+anyhow = "*"
+async-trait = "*"
+derive_more = "*"
+futures = "*"
+mockall = "*"
+must_future = "*"
+serde = "*"
+thiserror = "*"
```

Comment:
- can't be stars, but nix should flag that
- not sure it's cool to have both anyhow and thiserror - really wish there were just an obvious rust error type so we didn't have so many disparate approaches all mixed together

Reply:
Yeah, I'm hoping to make progress on #3098 so adding deps in new crates will be trivial, perhaps even in this crate